server/services/bedspaceSearchService.ts

```diff
@@ -32,6 +37,17 @@ export default class BedspaceSearchService {
       })
     ).sort((a, b) => a.name.localeCompare(b.name))
 
-    return { pdus }
+    const wheelchairAccessibilityData = filterCharacteristics(
```

Comment:
Maybe this this shoudl be called something like `bedspaceAttributes`. This would make it easier to understand as we have all bedspace attributes and then on line 45 we are explicitly filtering out the wheelchair attribute from the list